src/ansys/fluent/core/filereader/case.py

```diff
@@ -126,7 +126,7 @@ def __getattr__(self, name: str):
             return _CaseVariable(self._variables, name + "/")
 
 
-class Case:
+class CaseFile:
```

Comment:
maybe update the file name in line with this?

Reply:
Done, Thank you.